agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2890,6 +2935,74 @@ bool backhaul_manager::handle_1905_autoconfiguration_response(ieee1905_1::CmduMe
     return true;
 }
 
+bool backhaul_manager::handle_1905_beacon_metrics_query(ieee1905_1::CmduMessageRx &cmdu_rx,
+                                                        const std::string &src_mac,
+                                                        Socket *&forward_to)
+{
+    LOG(INFO) << "now going to handle BEACON METRICS QUERY";
+
+    // extract the desired STA mac
+    auto tlvBeaconMetricsQuery = cmdu_rx.getClass<wfa_map::tlvBeaconMetricsQuery>();
+    if (!tlvBeaconMetricsQuery) {
+        LOG(ERROR) << "handle_1905_beacon_metrics_query should handle only tlvBeaconMetrics, but "
+                      "got something else: 0x"
+                   << std::hex << (uint16_t)cmdu_rx.getMessageType();
+        return false;
+    }
+
+    const sMacAddr &requiredMac = tlvBeaconMetricsQuery->associated_sta_mac();
```

Comment:
We use Snake Case not Camel Case, so the name of the variable (as an example) should change to:
```suggestion
    const sMacAddr &required_mac = tlvBeaconMetricsQuery->associated_sta_mac();
```